### ISSUE TITLE
Webhook for new channel created

### ISSUE THREAD

**Issue by Huljo** [RocketChat/Rocket.Chat#5226]
Your Rocket.Chat version: 0.46

I need to know when a new channel is created but the outgoing webhooks feature seems to just handle watching of messages. (like Slack do)

If there is no other way to get an event when a new channel is created I'll add a webhook triggered by callback "afterCreateChannel" to handle it.

Edit: The webhooks/integrations feature is totally centered on messages events. Hard to believe I can just "add" channels events.

Please @RocketChat/core let me know if you think that webhooks for channels related events is a good/bad idea.

**Comment by engelgabriel** [comment]
@graywolf336 can you help here?

**Comment by graywolf336** [comment]
@engelgabriel This request is a new feature that we would need to add, which in the long term would be nice to have for Zapier so it gets notified instead of continually polls for new channels/groups. Is this something you're interested in?

**Comment by graywolf336** [comment]
@Huljo We will be adding this in the very near future 👍

**Comment by Huljo** [comment]
@engelgabriel Exactly what i'm interested in.

**Comment by engelgabriel** [comment]
@Huljo re-prioritized

**Comment by graywolf336** [comment]
Moving to the next release so that this lands around the same time as #5176

**Comment by Huljo** [comment]
Thanks guys

**Comment by Huljo** [comment]
Hi @graywolf336, I just tried the new "roomCreated" webhook but there is no post request when creating a new room... messages events work fine. :(

**Comment by graywolf336** [comment]
Did you turn on debugging and see what that outputs?

**Comment by Huljo** [comment]
Sorry, what do you mean by "tuning on debugging" ? :)

**Comment by Huljo** [comment]
@graywolf336, there is the logs when i create a new channel : 

````
[34mI20170403-12:12:35.244(2) [34mIntegrations ➔ Outgoing WebHook.debug Execute Trigger: roomCreated 
[34mI20170403-12:12:35.246(2) [34mIntegrations ➔ Outgoing WebHook.debug Got the event arguments for the event: roomCreated { event: 'roomCreated',   owner: { username: 'admin', _id: '35kRCc44rxsN9nRGF' },   room:     { name: 'test16',      t: 'c',      usernames: [ 'admin' ],      msgs: 0,      u: { _id: '35kRCc44rxsN9nRGF', username: 'admin' },      customFields: {},      ts: Mon Apr 03 2017 12:12:35 GMT+0200 (CEST),      ro: false,      sysMes: true,      _updatedAt: Mon Apr 03 2017 12:12:35 GMT+0200 (CEST),      _id: 'CuXN92trZSs8E5qSZ' } } 
[34mI20170403-12:12:35.247(2) [34mIntegrations ➔ Outgoing WebHook.debug Starting search for triggers for the room: CuXN92trZSs8E5qSZ 
[34mI20170403-12:12:35.247(2) [34mIntegrations ➔ Outgoing WebHook.debug Found 0 to iterate over and see if the match the event.
````

**Comment by Huljo** [comment]
Hi @graywolf336 any news about this?

**Comment by graywolf336** [comment]
@Huljo have you tried again with the latest version that was just released? I believe it was fixed there

**Comment by Huljo** [comment]
Thank you @graywolf336, problem solved in 0.55.1
